proto/packet.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+option go_package = "./pbpacket";
+
+package pbpacket;
+
+import "google/protobuf/any.proto";
+
+service Collector {
+  rpc Send (Packet) returns (CollectorReply) {}
+}
+
+// The request message containing the Packet
+message Packet {
+  google.protobuf.Any pcap = 1;
```

Comment:
if we decided to send multiple pcaps then we need to add `repeated` at the definition above

Reply:
Can we keep that for a followup improvment ? That raise more questions about the total size of the message.

The goal here was just to implement gRPC for security reasons to productize CLI.